src/components/fields/ServicePointAssignmentField.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Form } from 'react-final-form';
+import '../../test/__mocks__/matchMedia.mock';
+import ServicePointAssignmentField from './ServicePointAssignmentField';
+import withIntlConfiguration from '../../test/util/withIntlConfiguration';
+
+const servicePoints = [
```

Comment:
When possible, please use the shared test service points.

You can import them with:

```js
import ServicePoints from '../../test/data/ServicePoints';
```

Then, use them like `<ServicePointAssignmentField servicePoints={ServicePoints} />`

(You can also do `import * as ServicePoints from ...`, then do `ServicePoints.SERVICE_POINT_1` and such, if you need finer control over which service points get used)

Reply:
Thank you, pushed the changes.